Refresh.GameServer/Database/GameDatabaseContext.Levels.cs

```diff
@@ -348,6 +348,29 @@ public DatabaseList<GameLevel> SearchForLevels(int count, int skip, GameUser? us
     [Pure]
     public int GetTotalTeamPickCount(TokenGame game) => this._realm.All<GameLevel>().FilterByGameVersion(game).Count(l => l.TeamPicked);
 
+    [Pure]
+    public GameLevel? GetLevelByIdAndType(string slotType, int id)
+    {
+        GameLevel? level;
+        
+        switch (slotType)
+        {
+            case "user":
+                level = this.GetLevelById(id);
+                break;
+            case "developer":
+                if (id < 0)
+                    return null;
+                
+                level = this.GetStoryLevelById(id);
+                break;
+            default:
+                return null;
+        }
+        
+        return level;
```

Comment:
Don't see why you couldn't just return directly:
```suggestion
        switch (slotType)
        {
            case "user":
                return this.GetLevelById(id);
            case "developer":
                if (id < 0)
                    return null;
                
                return this.GetStoryLevelById(id);
            default:
                return null;
        }
```